src/components/Graph.react.js

```diff
@@ -322,12 +325,12 @@ const graphPropTypes = {
      * Plotly `figure` object. See schema:
      * https://plot.ly/javascript/reference
      * Only supports `data` array and `layout` object.
-     * `config` is set separately by the `config` property,
-     * and `frames` is not supported.
```

Comment:
You took out the note that `frames` is not supported... but it still says 
> Only supports `data` array and `layout` object.

Reply:
fixed.